plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -115,6 +124,30 @@ func (m *Ipmi) parse(acc telegraf.Accumulator, server string) error {
 		opts = conn.options()
 	}
 	opts = append(opts, "sdr")
+	if m.UseCache {
+		cacheFile := m.CachePath + "/" + server + "_ipmi_cache"
```

Comment:
Is this plugin meant for Linux only? I don't see any build tags or information in `README.md`.

But here I read that `ipmitool` can be run on Windows and MacOS: https://github.com/ipmitool/ipmitool/blob/master/INSTALL 
Does this plugin support this?

If yes, you should correct creating path - it should be OS independent.
If no, maybe this plugin should be built only for Linux?

@ssoroka @reimda what do you think?

Reply:
This plugin should work on any platform that provides the ipmitool program.  I'm not sure which platforms actually have the BMC hardware in practice but the ipmitool program is available on many platforms.  For example, stable debian ships it for x86, arm, mips, ppc64, s390.  (https://packages.debian.org/buster/ipmitool)  It's also available on windows and mac os.

With this in mind I think it's appropriate to leave out the build tags and let all telegraf platforms get the plug, and also require platform independent path manipulation in the code.

Reply:
sorry, was not aware that this tool is also working on windows and mac.
this should be fixed with 6c652b7